### ISSUE TITLE
Not loading on firefox or chromium ( works on chrome )

### ISSUE THREAD

**Issue by gtxaspec** [vbguyny/ws4kp#80]
Black screen when loading web site, both firefox and chromium.
```
firefox 105.0.1 console:

Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content. [node.js:393](resource://devtools/server/actors/inspector/node.js)
jQuery.Deferred exception: TwcQuery is null @https://battaglia.ddns.net/twc/Scripts/index.js?v=69:1389:5
j@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:29568
Deferred/then/g/</k<@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:29882
setTimeout handler*Deferred/then/g/<@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:30091
i@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:27603
fireWith@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:28369
fire@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:28405
i@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:27603
fireWith@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:28369
ready@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:31551
R@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:31685
EventListener.handleEvent*@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:31799
@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:220
@https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:225
 undefined [jquery-3.1.0.min.js:2:31145](https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js)
Uncaught TypeError: TwcQuery is null
    <anonymous> https://battaglia.ddns.net/twc/Scripts/index.js?v=69:1389
    jQuery 13
[index.js:1389:5](https://battaglia.ddns.net/twc/Scripts/index.js?v=69)
    <anonymous> https://battaglia.ddns.net/twc/Scripts/index.js?v=69:1389
    jQuery 13

​
chromium 106.0.5249.91:

jquery-3.1.0.min.js:2 jQuery.Deferred exception: Cannot read properties of null (reading 'replace') TypeError: Cannot read properties of null (reading 'replace')
    at HTMLDocument.<anonymous> (https://battaglia.ddns.net/twc/Scripts/index.js?v=69:1389:25)
    at j (https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:29568)
    at k (https://battaglia.ddns.net/twc/Scripts/jquery-3.1.0.min.js:2:29882) undefined
r.Deferred.exceptionHook @ jquery-3.1.0.min.js:2
k @ jquery-3.1.0.min.js:2
setTimeout (async)
(anonymous) @ jquery-3.1.0.min.js:2
i @ jquery-3.1.0.min.js:2
fireWith @ jquery-3.1.0.min.js:2
fire @ jquery-3.1.0.min.js:2
i @ jquery-3.1.0.min.js:2
fireWith @ jquery-3.1.0.min.js:2
ready @ jquery-3.1.0.min.js:2
R @ jquery-3.1.0.min.js:2
jquery-3.1.0.min.js:2 Uncaught TypeError: Cannot read properties of null (reading 'replace')
    at HTMLDocument.<anonymous> (index.js?v=69:1389:25)
    at j (jquery-3.1.0.min.js:2:29568)
    at k (jquery-3.1.0.min.js:2:29882)
(anonymous) @ index.js?v=69:1389
j @ jquery-3.1.0.min.js:2
k @ jquery-3.1.0.min.js:2
setTimeout (async)
r.readyException @ jquery-3.1.0.min.js:2
(anonymous) @ jquery-3.1.0.min.js:2
j @ jquery-3.1.0.min.js:2
k @ jquery-3.1.0.min.js:2
setTimeout (async)
(anonymous) @ jquery-3.1.0.min.js:2
i @ jquery-3.1.0.min.js:2
fireWith @ jquery-3.1.0.min.js:2
fire @ jquery-3.1.0.min.js:2
i @ jquery-3.1.0.min.js:2
fireWith @ jquery-3.1.0.min.js:2
k @ jquery-3.1.0.min.js:2
setTimeout (async)
(anonymous) @ jquery-3.1.0.min.js:2
i @ jquery-3.1.0.min.js:2
fireWith @ jquery-3.1.0.min.js:2
fire @ jquery-3.1.0.min.js:2
i @ jquery-3.1.0.min.js:2
fireWith @ jquery-3.1.0.min.js:2
ready @ jquery-3.1.0.min.js:2
R @ jquery-3.1.0.min.js:2
```

**Comment by vbguyny** [comment]
@gtxaspec the bug has been fixed and the site has been updated to v1.70
Thanks for letting me know about this!

**Comment by gtxaspec** [comment]
works as intended! thank you!
